### DISCUSSION TITLE
Προεραιτικό Θέμα ΓΕ_1

### DISCUSSION THREAD

**Discussion by IoannisTheodoropoulos** [sdy60/syllabus#98] [Q&A] [answered]
Καλημέρα σας, 
σχετικά με την παρουσίαση του προαιρετικού θέματος της Γραπτής Εργασίας, μέχρι πότε θα μπορούσαμε να δηλώσουμε οτι ενδιαφερόμαστε? Επίσης κατά την παράδοση της εργασίας θα πρέπει να παραδώσουμε και το συγκεκριμένο θέμα?

Ευχαριστώ!

**Comment by Asinapis13** [comment]
Καλημέρα, 
Το προαιρετικό θέμα της παρουσίασης πότε πρέπει να το δηλώσω

**Comment by Konssven** [comment, marked as answer]
Καλησπέρα

Ρώτησα τον Καθηγητή στην έναρξη της προηγούμενης ΟΣΣ και μου είπε ότι η δήλωση του προαιρετικού θέματος θα πρέπει να γίνει πριν την τελευταία ΟΣΣ. Αν δεν κάνω λάθος επειδή η τελευταία ΟΣΣ για αυτή την εργασία είναι αυτό το Σαββατοκύριακο η δήλωση θα πρέπει να γίνει πριν γίνει η συνάντηση. Ελπίζω να βοήθησα. και ελπίζω να μην κατάλαβα λάθος...

**Reply by epidrome** [reply]
Αν γίνει πρόσθετη ΟΣΣ1 πριν την ΓΕ1 τότε υπάρχει μια ακόμη ευκαιρία!
